projects/angular/src/emphasis/_labels.clarity.scss

```
@@ -110,84 +47,45 @@
   .label,
   a.label {
     //default
-    $label-nudge-for-vertical-alignment: variables.$clr_baselineRem_1px;
-    @include mixins.css-var(
-      font-size,
-      clr-label-font-size,
-      label-variables.$clr-label-font-size,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(
-      font-weight,
-      clr-label-font-weight,
-      label-variables.$clr-label-font-weight,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(
-      letter-spacing,
-      clr-label-letter-spacing,
-      label-variables.$clr-label-letter-spacing,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(
-      line-height,
-      clr-label-line-height,
-      // line-height needed to be adjusted to force center alignment
-      label-variables.$clr-label-font-size + variables.$clr_baselineRem_1px,
-      variables.$clr-use-custom-properties
-    );
-
+    @include mixins.generate-typography-token('CAPTION-LG-11');
     display: inline-flex;
     justify-content: center;
     align-items: center;
-    @include mixins.css-var(
-      padding,
-      clr-label-padding,
-      0 label-variables.$clr-label-padding-left-right 0,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(gap, clr-label-gap-size, 0, variables.$clr-use-custom-properties);
-    @include mixins.css-var(
-      border-radius,
-      clr-label-border-radius,
-      variables.$clr_baselineRem_0_5,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(
-      border-width,
-      clr-label-border-width,
-      label-variables.$clr-label-borderwidth,
-      variables.$clr-use-custom-properties
-    );
-    border-style: solid;
-    @include mixins.css-var(
-      border-color,
-      clr-label-default-border-color,
-      label-variables.$clr-label-default-border-color,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(
-      height,
-      clr-label-height,
-      variables.$clr_baselineRem_0_875,
-      variables.$clr-use-custom-properties
-    );
-    margin: 0 variables.$clr_baselineRem_0_25 $label-nudge-for-vertical-alignment 0;
+    gap: #{tokens.$cds-global-space-5};
+    padding: #{tokens.$cds-global-space-3} #{tokens.$cds-global-space-5};
+    border: label-variables.$clr-label-border-width solid label-variables.$clr-label-default-border-color;
+    border-radius: #{tokens.$cds-alias-object-border-radius-200};
+    height: label-variables.$clr-label-height;
```

Do we need $clr-label-height? Use token value directly.

```suggestion
    height: tokens.$cds-global-space-9;
```